playgrounds/all-field-types.mongodb

```diff
@@ -0,0 +1,42 @@
+const { Buffer } = require('node:buffer');
+
+use('test');
+
+// Create a document with all the BSON types.
+const doc = {
+  double: new Double(1.2), // Double, 1, double
+  string: 'Hello, world!', // String, 2, string
+  object: { key: 'value' }, // Object, 3, object
+  array: [1, 2, 3], // Array, 4, array
+  binData: new Binary(Buffer.from([1, 2, 3])), // Binary data, 5, binData
+  // Undefined, 6, undefined (deprecated)
+  objectId: new ObjectId(), // ObjectId, 7, objectId
+  boolean: true, // Boolean, 8, boolean
+  date: new Date(), // Date, 9, date
+  null: null, // Null, 10, null
+  regex: new BSONRegExp('pattern', 'i'), // Regular Expression, 11, regex
+  dbPointer: new DBRef('namespace', new ObjectId()), // DBPointer, 12, dbPointer (deprecated)
```

Comment:
Sooo … `DBPointer` and `DBRef` are different things :) `DBPointer` is an actual bson type, `DBRef` is a convention for regular BSON documents. The driver’s bson package doesn’t actually support `DBPointer`; when it encounters a `DBPointer` during deserialization, it converts it to `DBRef`.

(`DBRef` isn’t deprecated, either.)

Reply:
OK I'll make a PR to fix it. I want to add the two missing types anyway. Even though things are deprecated we should probably test to see what happens there.

Reply:
oh it hasn't been merged so I can just edit this one.